README.md

```diff
@@ -52,6 +50,8 @@ This allows the AMP library to include:
 * Some custom elements, called AMP HTML components, which make common patterns
   easy to implement in a performant way.
 
+Get started [creating your first AMP page](https://www.ampproject.org/docs/get_started/create_page.html).
```

Comment:
Left out the by, as the call is to get started creating the page, by changing the meaning a little bit.